.github/workflows/wiki_label.yml

```diff
@@ -0,0 +1,53 @@
+name: 'PR Wiki Label'
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  wiki-label:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && github.event.comment.body == '!wiki_label'
+    permissions:
+      contents: read
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check commenter authorization
+        id: check-auth
+        run: |
+          AUTHORIZED_USER="SynthTwo"
+          COMMENTER="${{ github.event.comment.user.login }}"
+          if [ "$COMMENTER" = "$AUTHORIZED_USER" ]; then
+            echo "authorized=true" >> $GITHUB_OUTPUT
+          else
+            echo "authorized=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Add Wiki label and thumbs up reaction
+        if: steps.check-auth.outputs.authorized == 'true'
+        run: |
+          gh pr edit ${{ github.event.issue.number }} --add-label "Requires Wiki Update"
+          echo "Label 'Requires Wiki update' added to PR #${{ github.event.issue.number }}"
+          curl -X POST \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            "https://api.github.com/repos/${{ github.repository }}/issues/comments/${{ github.event.comment.id }}/reactions" \
+            -d '{"content":"+1"}'
+          gh pr comment ${{ github.event.issue.number }} --body "Label 'Requires Wiki update' has been added."
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Unauthorized user comment and thumbs down reaction
+        if: steps.check-auth.outputs.authorized == 'false'
+        run: |
+          gh pr comment ${{ github.event.issue.number }} --body "Sorry, you are not authorized to use the !wiki_label command."
+          curl -X POST \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            "https://api.github.com/repos/${{ github.repository }}/issues/comments/${{ github.event.comment.id }}/reactions" \
+            -d '{"content":"-1"}'
```

Comment:
I think this can be removed and only post a comment if the user is authorized.

Reply:
Would you be amenable to removing the comment posting, but leaving the thumbs down reaction if rejected?

Reply:
IMO, less comment bloat the better, I think reactions are good enough for both the authorized and unauthorized cases

Reply:
I can agree with that. I wasn't really sure about adding the additional messages to begin with. Considering as soon as the bot does accept a valid command, it will just apply the label and thumbs up the comment. Plus, I don't really see the need now for rejection message either, since the authorized user pool will be small regardless.

Reply:
A thumbsup on the original message is better. Or just leave out the comments altogether since indeed they will apply the change requested as a response

Reply:
Yup! I am just going to remove the comments as suggested, and leave the thumbs up and down. I think that is best.